docker-compose-test.yml

```diff
@@ -12,6 +12,21 @@ services:
       - POSTGRES_PASSWORD=form-builder
       - POSTGRES_USER=form-builder
       - POSTGRES_DB=form-builder
+    command: ["postgres", "-c", "log_statement=all", "-c", "log_destination=stderr"]
+
+  form-db-unit-tests:
+    image: postgres
+    networks:
+      - hmpps
+    container_name: form-builder-db-unit-tests
+    restart: always
+    ports:
+      - "5434:5432"
+    environment:
+      - POSTGRES_PASSWORD=form-builder-unit-tests
+      - POSTGRES_USER=form-builder-unit-tests
+      - POSTGRES_DB=form-builder-unit-tests
+    command: ["postgres", "-c", "log_statement=all", "-c", "log_destination=stderr"]
```

Comment:
Can either create a new DB for unit testing, or we can manually `create database form-builder-unit-test` inside the existing DB, or we can create an init script for the existing DB to create 2 (or 3, one for risk-profiler) databases on first boot?